docs/architecture/README.md

```diff
@@ -0,0 +1,40 @@
+# Architecture Decision Records (ADR)
+
+This is a location to record all high-level architecture decisions in the Cosmos-SDK.
+
+An Architectural Decision (**AD**) is a software design choice that addresses a functional or non-functional requirement that is architecturally significant.
+An Architecturally Significant Requirement (**ASR**) is a requirement that has a measurable effect on a software system’s architecture and quality.
+An Architectural Decision Record (**ADR**) captures a single AD, such as often done when writing personal notes or meeting minutes; the collection of ADRs created and maintained in a project constitute its decision log. All these are within the topic of Architectural Knowledge Management (AKM).
+
+You can read more about the ADR concept in this [blog post](https://product.reverb.com/documenting-architecture-decisions-the-reverb-way-a3563bb24bd0#.78xhdix6t).
```

Comment:
```suggestion
You can read more about the ADR concept in the [Documenting architecture decisions, the Reverb way](https://product.reverb.com/documenting-architecture-decisions-the-reverb-way-a3563bb24bd0#.78xhdix6t) blog post.
```

Reply:
@PaddyMc we have non-descriptive link text all over our technical content, but our readers benefit from using descriptive link text https://developers.google.com/style/link-text

Reply:
@SenorMonito you can see I suggested changes to the link text that include the blog article title... we have so many links that say [here] which are literally not helpful. (hi David!)